src/util/subprocess.h

```diff
@@ -1366,6 +1375,11 @@ namespace detail {
       util::write_n(err_wr_pipe_, err_msg.c_str(), err_msg.length());
     }
 
+#ifdef CODE_COVERAGE_ENABLED
+      __gcov_dump();
+      __gcov_reset();
```

Comment:
These lines are indented too much compared to surrounding code.

Are they necessary at all?

Reply:
Well, reset seem unneeded here. I'm going to drop it.